astrology.lic

```diff
@@ -2,19 +2,25 @@
   Documentation: https://elanthipedia.play.net/Lich_script_repository#astrology
 =end
 
-custom_require.call(%w(common common-arcana drinfomon spellmonitor))
+custom_require.call(%w(common common-arcana common-travel drinfomon spellmonitor))
 
 class Astrology
   include DRC
   include DRCA
+  include DRCT
 
   def initialize
     settings = get_settings
     @constellations = get_data('constellations').constellations
     @have_telescope = settings.have_telescope
+    @ap_training = settings['astral_plane']['train_in_ap']
+    @ap_source = settings['astral_plane']['train_source']
```

Comment:
I could rename it to `astral_plane_training` and move the `exit_if_killed` outside. `exit_if_killed` would have to be renamed to something more clear like `quit_on_death_in_ap`